Dockerfile

```diff
@@ -13,8 +13,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 # Copy the bot code into the container.
 COPY . .
 # Run app.py when the container launches
-CMD ["python3", "main.py"]
-
-
-# To run the application in Docker: docker run -it --rm --name my-running-app my-python-app
```

Comment:
Don't know if both of these lines were meant to be removed as well, but I don't user docker...

Reply:
(Lines 19 + 20)

Reply:
I removed them since I implemented a docker build and run files to streamline the build and run process so I didn't have to hardcode anything. Those commands weren't the right ones that needed to be used anyways!